### PR TITLE
Use pty.js instead of child_pty.

### PR DESCRIPTION
**Pull request by shockone**
* Use pty.js instead of child_pty.
* Simplify the build command.

### CODE REVIEW

**Comment by shockone** [comment]
@black-screen/team please review.

**Comment by G07cha** [comment]
I planned to test tomorrow on the OS X

**Comment by shockone** [comment]
I imagined a code review to be a review of only the code quality, irrelatively to whether it works. :)

**Comment by G07cha** [comment]
Sure I've looked on a code and it's seems fine so I'll do my best, test if it's working.

File: src/Invocation.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /// <reference path="references.ts" />
 
-var child_pty = require('child_pty');
+var pty = require('pty.js');
```
Comment (RobertoUa) [inline src/Invocation.ts]: why not import?

Comment (shockone) [inline src/Invocation.ts]: There is no type definition for this module. It won't compile with `import`.

Comment (RobertoUa) [inline src/Invocation.ts]: is it hard to add the type definition with one 'spawn' method?

Comment (shockone) [inline src/Invocation.ts]: Not really, similar to this https://github.com/black-screen/black-screen/blob/master/dts/AnsiParser.d.ts. Will you? :)

**Comment by RobertoUa** [comment]
awesome!

**Comment by shockone** [comment]
@RobertoUa, I know, right?

**Pull request merged**